packages/common-all/src/types/typesv2.ts

```diff
@@ -364,6 +365,8 @@ export type GetDecorationsOpts = {
 export type DCommonProps = {
   /** Dictionary where key is the note id. */
   notes: NotePropsDict;
+  /** Dictionary where the key is lowercase note fname, and values are ids of notes with that fname. */
```

Comment:
nit: would you mind expanding on this one a bit saying that the reason there can be multiple id's is because of multi-vault?